dashboard/cron.py

```diff
@@ -42,221 +41,35 @@
 # Imports the Google Cloud client library
 from google.cloud import bigquery
 
+from django_cron import CronJobBase, Schedule
+
 
 logger = logging.getLogger(__name__)
 
-# ## Connect to Student Dashboard's MySQL database
-#
-# ### Get enrolled users and files within site
 db_name = settings.DATABASES['default']['NAME']
 db_user = settings.DATABASES['default']['USER']
 db_password = settings.DATABASES['default']['PASSWORD']
 db_host = settings.DATABASES['default']['HOST']
 db_port = settings.DATABASES['default']['PORT']
 
-engine = create_engine("mysql+pymysql://{user}:{password}@{host}:{port}/{db}"
```

Comment:
create_engine is duplicated a bunch of times in this code. Were you not able to re-use this engine?

Reply:
Done! Moved out from class and defined as global variable.